packages/nodes-base/credentials/UProcApi.credentials.ts

```diff
@@ -19,4 +25,18 @@ export class UProcApi implements ICredentialType {
 			default: '',
 		},
 	];
+	async authenticate(
+		credentials: ICredentialDataDecryptedObject,
+		requestOptions: IHttpRequestOptions,
+	): Promise<IHttpRequestOptions> {
+		const token = Buffer.from(`${credentials.email}:${credentials.apiKey}`).toString('base64');
+		requestOptions.headers!['Authorization'] = `Basic ${token}`;
+		return requestOptions;
+	}
+	test: ICredentialTestRequest = {
+		request: {
+			baseURL: 'https://api.uproc.io/api/v2/process',
```

Comment:
uProc credential test does not seem to work since /me does not exist (returns 404)